BasicSample/Assets/ARAnchor/ARAnchor.unity

```diff
@@ -617,12 +617,15 @@ TextMesh:
   m_PrefabInstance: {fileID: 0}
   m_PrefabAsset: {fileID: 0}
   m_GameObject: {fileID: 664740217}
-  m_Text: "AR Anchors and Anchor Persistence\n\nAir tap anywhere to create a free-world
-    anchor.\nAir tap near an anchor to toggle persistence.\nClear Anchor Store removes
-    persistence from all \nanchors\nClear Scene Anchors removes all anchors from
-    \nscene but maintains persistence"
+  m_Text: "AR Anchors and Anchor Persistence\n\n- Air tap anywhere to create an AR
+    Anchor, represented as green cube.\n- Air tap at an anchor's cube to toggle anchor
```

Comment:
I think you got the blue and green cubes mixed up. 
Blue - unpersistent
Green - persistent